.gitignore

```diff
@@ -133,3 +133,7 @@ dmypy.json
 
 # Persistent data file
 data/proteins.json
+
+# Custom - backend
+backend/embeddings/**/**
+backend/data/pdb_data_seq.csv
```

Comment:
We never want to commit data, as a rule of thumbs. So maybe the following works?

```suggestion
backend/data/*
```

Reply:
Oh! I did not `data` is not to be committed. It currently is at commit a7032bd37ef60bd5b6d797a4da2e92cbdd37231b of `main`, and is also not added in the `.gitignore`.

Reply:
Yes I know, that has been handy with some toy data. But we can now ignore the data folder I would say?

Reply:
Yeah I agree let's remove all the toy data moving forward.

Reply:
Has this been reverted?